soc/arm/st_stm32/stm32l5/soc.c

```diff
@@ -47,6 +47,11 @@ static int stm32l5_init(const struct device *arg)
 	LL_APB1_GRP1_EnableClock(LL_APB1_GRP1_PERIPH_PWR);
 	LL_PWR_SetRegulVoltageScaling(LL_PWR_REGU_VOLTAGE_SCALE0);
 
+#if defined(PWR_CR3_UCPD_DBDIS)
+	/* Disable USB Type-C dead battery pull-down behavior */
+	__HAL_RCC_PWR_CLK_ENABLE();
```

Comment:
LL_APB1_GRP1_EnableClock(LL_APB1_GRP1_PERIPH_PWR); is called just before.

Reply:
removed